website/docs/core/applications.md

```diff
@@ -54,3 +54,13 @@ Requires authentik 2022.3
 :::
 
 To give users direct links to applications, you can now use an URL like `https://authentik.company/application/launch/<slug>/`. This will redirect the user directly if they're already logged in, and otherwise authenticate the user, and then forward them.
+
+### Backchannel providers
+
+:::info
+Requires authentik 2023.5
```

Comment:
```suggestion
Requires authentik version 2023.5 or later.
```

Reply:
We should probably go through all version notes and bring them in the same format